tests/test___main__.py

```diff
@@ -21,27 +21,40 @@
 import os
 import subprocess
 
+import pytest
+
 # pylint: disable=redefined-outer-name, invalid-name
 # pylint: disable=missing-class-docstring, missing-function-docstring
 # pylint: disable=too-many-arguments, unused-argument, too-few-public-methods
 
 
+python_path = ("__init__.py",)
+cpp_path = ("include", "p_p", "p_p.hpp")
```

Comment:
[consider] Is it worth using the 'dict keyed by generator' approach here too for consistency?

Reply:
Switched to a dict in 1e1ad134

Reply:
Ahh I was imagining you'd key it with `generator` like https://github.com/OpenAssetIO/OpenAssetIO-TraitGen/blob/1e1ad13476946e431a7ab119c11206673ed52657/tests/test___main__.py#L80, but no biggie!

Reply:
Oh no, of course, that's a good idea.

Reply:
Simplified in 7139867e